build.sbt

```diff
@@ -58,10 +58,17 @@ libraryDependencies ++= Seq(
 	"com.typesafe.akka"    %% "akka-actor"          % akkaVersion % Provided,
 	"com.rabbitmq"         % "amqp-client"          % "5.6.0",
 	"com.typesafe.akka"    %% "akka-testkit"        % akkaVersion  % Test,
-	"org.scalatest"        %% "scalatest"           % "3.0.5" % Test,
+	"org.scalatest"        %% "scalatest"           % "3.0.8" % Test,
 	"junit"                % "junit"                % "4.12" % Test,
 	"com.typesafe.akka"    %% "akka-slf4j"          % akkaVersion % Provided,
 	"ch.qos.logback"       %  "logback-classic"     % "1.0.0" % Provided
 )
 
+libraryDependencies ++= (CrossVersion.partialVersion(scalaVersion.value) match {
+	case Some((2, scalaMajor)) if scalaMajor < 13 =>
+	  Seq("org.scala-lang.modules" %% "scala-collection-compat" % "2.1.1")
```

Comment:
Nice, I did not know about this. We could upgrade safe-config to fix this: https://github.com/gawkermedia/safe-config/pull/11/files#diff-3638d2f3d2747abc3c00823c75656c31R7